src/main/java/tr/com/orioninc/laborant/service/LabService.java

```diff
@@ -106,27 +106,27 @@ public String getALlLabsStatus(){
                             }
                         }
                         if (failFound){
-                            outputString +="SIGNAL 2";
+                            outputString.append("SIGNAL 2");
                         }
                         else if (!failFound && stopFound) {
-                            outputString+="SIGNAL 1";
+                            outputString.append("SIGNAL 1");
                         }
                         else {
-                            outputString+="SIGNAL 0";
+                            outputString.append("SIGNAL 0");
                         }
                     }
-                    outputString+=  " \n";
-                    outputString+=response;
-                    outputString+="\n  \n";
+                    outputString.append(" \n");
+                    outputString.append(response);
+                    outputString.append("\n  \n");
                 }
                 catch(Exception e)
                 {
                     e.printStackTrace();
```

Comment:
Print this stack trace with logger in ERROR level:
`log.error("[getALlLabsStatus] exception while bla bla", e);`

Reply:
Ditto